pkg/server/config/config.go

```diff
@@ -121,3 +131,19 @@ func BuildTLS(opts TLS) (*tls.Config, error) {
 
 	return config, nil
 }
+
+func (ap *AuthenticatingProxy) fixSpiffeIds() []int {
+	var fixedSpiffeIds []spiffeid.ID
+	var failedSpiffeIds []int
+
+	// Find if spiffe IDs failed to report them
+	for index, value := range ap.AllowedSpiffeIds {
+		if !value.IsZero() {
```

Comment:
more for my own understanding, are the spiffe ids automatically parsed and if it can't be parsed correctly, it returns empty?